### PR TITLE
First version

### PR DESCRIPTION
**Pull request by lonsmin**
**Todo app  **

- add
- del
- more items

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,26 +1,90 @@
...
+      return data
+    },
+    async moreItems(){
+      this.params.pageNumber = this.params.pageNumber + 1;
+      this.tasks = this.tasks.concat(await this.getTasks())
+    }
+  },
+  computed: {
+    
+  },
```
Comment (katemihalikova) [inline src/App.vue:68]: `computed` is empty, remove

File: src/components/Header.vue [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<template>
+    <header>
+        <h1>{{title}}</h1>
+        <Button color="green" text="Add"/>
```
Comment (katemihalikova) [inline src/components/Header.vue]: What does this button do?

File: src/components/subcomponents/Button.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            default:'Tlačítko'
+        },
+        color:{
+            type:String,            
+        },
+       
+    },
+    methods:{
+        loadPost(){
+            console.log('loading...')
```
Comment (katemihalikova) [inline src/components/subcomponents/Button.vue]: Remove `console.log`. Then, `loadPost` will be empty, so remove it as well.

File: src/components/subcomponents/Task.vue [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+.task{
+    background: #f4f4f4;
+    border-radius: 5px;
+    margin: 5px;
+    padding: 10px 20px;
+}
+p{
+    margin: 5px 0;
+}
+</style>
```
Comment (katemihalikova) [inline src/components/subcomponents/Task.vue:40]: Nice simple component 👍

**Review by katemihalikova — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (lonsmin) [reply src/App.vue:68]: removed

File: src/components/Header.vue (hunk 2 above)
Comment (lonsmin) [reply src/components/Header.vue]: This is toggle button. Still not fully functional
- commented

File: src/components/subcomponents/Button.vue (hunk 3 above)
Comment (lonsmin) [reply src/components/subcomponents/Button.vue]: Removed also with methods.

File: src/components/subcomponents/Task.vue (hunk 4 above)
Comment (lonsmin) [reply src/components/subcomponents/Task.vue:40]: Thank you :relaxed:

**Comment by lonsmin** [comment]
## I was complete all the required changes
See [app](https://lonsmin.github.io/Vue_posts_api/).

**Review by katemihalikova — APPROVED** [review]
Everything ok :+1:

**Pull request merged**